### PR TITLE
Add config to client get_status dislay.

### LINKED ISSUE #2071: Missing algo config in get_status

### PR DESCRIPTION
**Pull request by AureliaDolo**
Fix #2071

### CODE REVIEW

File: massa-models/src/settings.rs [hunk 1]
```diff
@@ -41,3 +43,42 @@ pub struct CompactConfig {
...
+        writeln!(f, "    Thread count: {}", self.thread_count)?;
+        writeln!(f, "    t0: {}", self.t0)?;
+        writeln!(f, "    delta_f0: {}", self.delta_f0)?;
+        writeln!(
+            f,
+            "    Operation validity periods: {}",
+            self.operation_validity_periods
+        )?;
+        writeln!(f, "    Periods per cycle: {}", self.periods_per_cycle)?;
+
```
Comment (yvan-sraka) [inline massa-models/src/settings.rs:66]: Why all these spaces?

**Review by yvan-sraka — APPROVED** [review]

File: massa-models/src/settings.rs (hunk 1 above)
Comment (AureliaDolo) [reply massa-models/src/settings.rs:66]: For the get_status command to look nice in the interactive client.

It's the workaround we're using everywhere else :/

Comment (yvan-sraka) [reply massa-models/src/settings.rs:66]: I'm talking about empty lines in code, but never mind...

Comment (AureliaDolo) [reply massa-models/src/settings.rs:66]: Ow I can fix it :)

**Review by yvan-sraka — APPROVED** [review]
LGTM

**Comment by AureliaDolo** [comment]
bors merge

**Comment by AureliaDolo** [comment]
bors merge

**Comment by AureliaDolo** [comment]
bors merge

**Pull request merged**